orders/views.py

```diff
@@ -1,3 +1,21 @@
-# from django.shortcuts import render
+from django.contrib.auth.decorators import login_required
+from django.shortcuts import render
 
-# Create your views here.
+from dogowner.models import DogOwner
+from orders.models import Order
+
+
+@login_required()
+def orders(request):
+    if DogOwner.objects.filter(user=request.user).exists():
+        context = {
+            'orders': Order.objects.filter(dog_owner_id__user=request.user),
```

Comment:
Why is it necessary to fillter by authenticated user, is it possible to use `request.user` directly?